### ISSUE TITLE
No package for v0.35.0 on Ubuntu 22.04

### ISSUE THREAD

**Issue by jamielinux** [vectordotdev/vector#19678]
### A note for the community

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Problem

I'm not sure what the usual interval is between your GitHub Release and when your Ubuntu package becomes available, but I suspect it's usually not more than a week. `v0.35.0` was released 2 weeks ago and I've just noticed that there's not yet a package available for Ubuntu 22.04 from your repository. 

If the interval usually is this long, then just ignore me. But thought I'd give the heads up in case there was some problem with your package repository pipeline.

```console
# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy

# date
Mon Jan 22 09:07:52 AM UTC 2024

# apt-get update -q
Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease
Hit:2 http://de.archive.ubuntu.com/ubuntu jammy InRelease
Hit:3 https://download.docker.com/linux/ubuntu jammy InRelease
Hit:4 http://de.archive.ubuntu.com/ubuntu jammy-updates InRelease
Get:5 https://repositories.timber.io/public/vector/deb/ubuntu jammy InRelease [4,947 B]
Hit:6 http://de.archive.ubuntu.com/ubuntu jammy-backports InRelease
Fetched 4,947 B in 1s (9,179 B/s)
Reading package lists...

# apt-cache policy vector | head -n 10
vector:
  Installed: 0.32.1-1
  Candidate: 0.34.2-1
  Version table:
     0.34.2-1 500
        500 https://repositories.timber.io/public/vector/deb/ubuntu jammy/main amd64 Packages
     0.34.1-1 500
        500 https://repositories.timber.io/public/vector/deb/ubuntu jammy/main amd64 Packages
     0.34.0-1 500
        500 https://repositories.timber.io/public/vector/deb/ubuntu jammy/main amd64 Packages
```

### Version

v0.34.2

**Comment by jszwedko** [comment]
Hey @jamielinux !

This release, v0.35.0, was actually the first release only published to the new apt repositories. This was mentioned in the release notes here: https://vector.dev/releases/0.35.0/.  There was a page published with the v0.34.0 that goes into more detail: https://vector.dev/highlights/2023-11-07-new-linux-repos/. The TL;DR is that you should use `apt.vector.dev` instead of `repositories.timber.io`.

**Comment by jamielinux** [comment]
@jszwedko Oops! So sorry for the noise :flushed:
